source/installation/file-location.txt

```diff
@@ -0,0 +1,80 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides the location of files needed when working with 
+Relational Migrator. The location of the files varies according to the OS 
```

Comment:
(wording -- minimalism)
Tweaking fit a bit better with the rewrite to the first sentence and for improved minimalism. 

```suggestion
Relational Migrator. Their location varies according to the OS 
```